client/my-sites/themes/themes-banner/index.jsx

```diff
@@ -46,6 +50,12 @@ class ThemesBanner extends PureComponent {
 		recordTracksEvent( 'calypso_showcase_banner_click', tracksData );
 	};
 
+	// eslint-disable-next-line no-undef
+	handleBannerClose = e => {
+		this.props.hideThemesBanner();
+		e.preventDefault();
```

Comment:
This doesn't seem to work -- clicking on the X doesn't just close the banner for me, but also redirects me to the theme advertised by the banner.

Reply:
This still seems to be happening for me in logged-out mode, but we hopefully fix that separately.